Cargo.toml

```diff
@@ -40,15 +37,20 @@ thiserror = "1.0"
 tokio-stream = "0.1"
 nix = "0.23.0"
 libc = "0.2"
+getrandom = { version = "0.2", optional = true }
 
 [dependencies.tokio]
 version = "1.14"
 features = ["fs", "io-util", "sync", "time", "rt", "macros", "rt-multi-thread"]
 
 [features]
 # default = ["benchmark"]
+default = ["runtime-rng"]
```

Comment:
We don't need `runtime-rng` and everything related to this feature. Please, remove it